iFixit/react-commerce
# Vulcan: Make Solution card look like mockup

This should be mostly matching the mockup at this point.

## QA Notes
Not all the wikitext is formatted nicely yet; please ignore how ugly bulleted lists and links look for now.

Also, the mockup includes more parts that we're not supporting yet; I made a bunch of the extra things invisible on it to get a feel for what this should look like without upvotes or card footers. Please comment if something looks weird; it's kinda scrawny right now because of the lack of features.

You'll need to add
```
[comment]solutions[/comment]
```
markup to mark where the solutions begin on your test page. You can also add
```
[comment]conclusion[/comment]
```
to mark the place where the solutions end.

Example: https://www.cominor.com/Wiki/Dryer_Making_Loud_Noise
Append: `/Vulcan/Dryer_Making_Loud_Noise`

Closes iFixit/ifixit#46092
